### ISSUE TITLE
Add PDBFixer integration

### ISSUE THREAD

**Issue by jchodera** [peastman/openmm-setup#7]
If you let people load PDB files but don't even warn them that nearly all PDB files downloaded from the RCSB are unusable as input without further processing, people are going to get frustrated and give up.

**Comment by peastman** [comment]
Yes, that's just another thing I haven't gotten to yet. :)  It's on my list!
